src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/mapper/JJDisputeMapper.java

```diff
@@ -0,0 +1,143 @@
+	@Mapping(source = "icbcReceivedDt", target = "icbcReceivedDate")
+	@Mapping(source = "disputantBirthDt", target = "contactInformation.birthdate")
+	@Mapping(source = "disputantDrvLicNumberTxt", target = "contactInformation.driversLicenceNumber")
+//	@Mapping(source = "drvLicIssuedProvSeqNo", target = "contactInformation.province") // TODO: field missing in model but exists in database
+//	@Mapping(source = "drvLicIssuedCtryId", target = "drvLicIssuedCtryId")             // TODO: field missing in model but exists in database
+	@Mapping(source = "emailAddressTxt", target = "contactInformation.emailAddress")
+	@Mapping(source = "addressLine1Txt", target = "contactInformation.address")
+	@Mapping(source = "lawyerSurnameNm", target = "lawyerSurname")
+	@Mapping(source = "lawyerGiven1Nm", target = "lawyerGivenName1")
+	@Mapping(source = "lawyerGiven2Nm", target = "lawyerGivenName2")
+	@Mapping(source = "lawyerGiven3Nm", target = "lawyerGivenName3")
+	@Mapping(source = "lawFirmNm", target = "lawFirmName")
+	@Mapping(source = "interpreterLanguageCd", target = "interpreterLanguageCd")
+	@Mapping(source = "courtHearingTypeCd", target = "hearingType", qualifiedByName="mapHearingType")
+	@Mapping(source = "disputeCounts", target = "jjDisputedCounts")
+	@Mapping(source = "entDtm", target = "createdTs")
+	@Mapping(source = "entUserId", target = "createdBy")
+	@Mapping(source = "updDtm", target = "modifiedTs")
+	@Mapping(source = "updUserId", target = "modifiedBy")
+	public abstract JJDispute convert(ca.bc.gov.open.jag.tco.oracledataapi.ords.tco.api.model.JJDispute jjDispute);
+
+	@Mapping(source = "disputeCountId", target = "id")
+	@Mapping(source = "countNo", target = "count")
+//	@Mapping(source = "statuteId", target = "statuteId")                                // TODO: field missing in model but exists in database
+	@Mapping(source = "pleaCd", target = "plea", qualifiedByName="mapPlea")
+	@Mapping(source = "fineDueDt", target = "dueDate")
+//	@Mapping(source = "violationDt", target = "violationDt")                            // TODO: field missing in model but exists in database
+	@Mapping(source = "adjustedAmt", target = "lesserOrGreaterAmount")
+	@Mapping(source = "includesSurchargeYn", target = "includesSurcharge")
+	@Mapping(source = "revisedDueDt", target = "revisedDueDate")
+	@Mapping(source = "totalFineAmt", target = "totalFineAmount")
+	@Mapping(source = "commentsTxt", target = "comments")
+	@Mapping(source = "requestTimeToPayYn", target = "requestTimeToPay")
+	@Mapping(source = "requestReductionYn", target = "requestReduction")
+	@Mapping(source = "requestCourtAppearanceYn", target = "appearInCourt")
+	@Mapping(source = "entDtm", target = "createdTs")
+	@Mapping(source = "entUserId", target = "createdBy")
+	@Mapping(source = "updDtm", target = "modifiedTs")
+	@Mapping(source = "updUserId", target = "modifiedBy")
+	public abstract JJDisputedCount convert(ca.bc.gov.open.jag.tco.oracledataapi.ords.tco.api.model.JJDisputeCount jjDisputeCount);
```

Comment:
Note: there is a warning in the code here which I left intentionally so these fields will get dealt with some day. These fields are not in ORDS and so have no map to the main application.

`Unmapped target properties: "description, jjDispute, jjDisputedCountRoP, ticketedFineAmount".`

Reply:
Done.